settlement/settlement.go

```diff
@@ -41,11 +42,81 @@ type Transaction struct {
 	SettlementID     string                   `json:"transactionId" valid:"uuidv4"`
 	TransferFee      decimal.Decimal          `json:"fee"`
 	Type             string                   `json:"type"`
-	ValidUntil       time.Time                `json:"validUntil"`
+	ValidUntil       time.Time                `json:"validUntil,omitempty"`
 	DocumentID       string                   `json:"documentId,omitempty"`
 	Note             string                   `json:"note"`
 }
 
+// AntifraudTransaction a transaction object that comes from antifraud
+type AntifraudTransaction struct {
```

Comment:
Can any of these be nullable? Are we worried about that?

Reply:
i'm not so worried about it, but perhaps @nvonpentz can shed some light on what to expect from antifraud output